show/main.py

```diff
@@ -966,6 +966,44 @@ def counters(ctx, verbose, period, printall):
 
         run_command(cmd, display_cmd=verbose)
 
+# 'errors' subcommand ("show interfaces counters errors")
+@counters.command()
+@click.option('-a', '--printall', is_flag=True)
+@click.option('-p', '--period')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+@click.pass_context
+def errors(ctx, verbose, period, printall):
+    """Show interface counters errors"""
+
+    if ctx.invoked_subcommand is None:
```

Comment:
Why checking for invoked_subcommand? This is a command, not a group; there cannot be subcommands.

Reply:
OK I've updated that.  I was copying what "show interface counter rif" did.  Removed subcommand section.